src/test/java/emissary/util/shell/ReadOutputBufferTest.java

```diff
@@ -0,0 +1,29 @@
+package emissary.util.shell;
+
+import emissary.test.core.junit5.UnitTest;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+
+import java.io.InputStream;
+import java.nio.charset.Charset;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
(nit) replace wildcard import
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;
import static org.junit.jupiter.api.Assertions.assertNull;
import static org.junit.jupiter.api.Assertions.fail;
```